### PR TITLE
Add account table entity

### LINKED ISSUE #18: Add Account table/entity
1. Create Liquibase changeset for Account table:
* id
* number
* user_id (foreign key)
* balance
* 
2. Create the Account entity with the fields mentioned above.

### PR DESCRIPTION
**Pull request by andy-sam**
Fixes #18

### CODE REVIEW

File: src/main/resources/db/changelog/changelog-2021-06-24-01.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-table-user
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-06-24-01.yaml:3]: fix id

File: src/main/resources/db/changelog/changelog-2021-06-24-01.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: pk_account
+                    nullable: false
+              - column:
+                  name: number
+                  type: varchar(512)
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-06-24-01.yaml:24]: make it 16 symbols

File: src/main/resources/db/changelog/changelog-2021-06-24-01.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_account_user
+                    references: user(id)
+              - column:
+                  name: balance
+                  type: BIGINT
+                  constraints:
+                    nullable: true
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/changelog-2021-06-24-01.yaml:38]: 1. default value = 0
2. add constraint not less than 0

**Review by alex-anakin — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changelog-2021-06-24-01.yaml (hunk 1 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/changelog-2021-06-24-01.yaml:3]: done

File: src/main/resources/db/changelog/changelog-2021-06-24-01.yaml (hunk 3 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/changelog-2021-06-24-01.yaml:38]: done

**Review by alex-anakin — APPROVED** [review]

**Pull request merged**